department-of-veterans-affairs/caseflow
# DAS Checkbox Case Timeline and Math: APPEALS-11839, APPEALS-11841, APPEALS-11842

Resolves
https://vajira.max.gov/browse/APPEALS-11839
https://vajira.max.gov/browse/APPEALS-11841
https://vajira.max.gov/browse/APPEALS-11842

### Description
Please explain the changes you made here.
- Code refactor and reorganization: due to nature of changes and incoming changes related to doing this work for Legacy better code organization was needed for this area of Queue's Evaulate Decision page
- Change DAS Case Timeline display (see images below)
- Update math for DAS Case Timeliness to remove +1 adjustment total and account for days when a case is back with a judge for days worked

### Acceptance Criteria

### Testing Plan
1. On a DEMO env
2. Login in as BVAEBECKER
3. Go to Your Queue
4. Pick either case
5. In actions dropdown select Ready for dispatch
6. Navigate through screens until reaching the Evaluate Decision page
7. Scroll down to Case Timeliness section
8. Observe changes

#### Test plans
- https://vajira.max.gov/browse/APPEALS-11247
- https://vajira.max.gov/browse/APPEALS-11270
- https://vajira.max.gov/browse/APPEALS-11382
- https://vajira.max.gov/browse/APPEALS-11407
- https://vajira.max.gov/browse/APPEALS-11409

 BEFORE|AFTER
 ---|---
![Screen Shot 2022-11-25 at 11 45 13 AM](https://user-images.githubusercontent.com/1094999/204028490-0a6fab9a-5a53-4369-98ea-1cca7885fb30.png)
![Screen Shot 2022-11-25 at 11 44 42 AM](https://user-images.githubusercontent.com/1094999/204028495-8ef64824-64b5-45ac-922d-11b5f7b3d951.png)
